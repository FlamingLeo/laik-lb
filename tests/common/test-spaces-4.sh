#!/bin/sh
${LAUNCHER-./launcher} -n 4 ../src/spacestest | LC_ALL='C' sort > test-spaces-4.out
cmp test-spaces-4.out "$(dirname -- "${0}")/test-spaces-4.expected"
