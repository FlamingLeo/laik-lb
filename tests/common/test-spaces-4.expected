T0: 1d-space: -100 - 1000
T0: 1d-space: 0 - 100
T0: 2d-space: -200 - 2000 / -300 - 3000
T0: 2d-space: 0 - 200 / 0 - 300
T0: 3d-space: 0 - 400 / 0 - 500 / 0 - 600
T0: 3d-space: 0 - 400 / 0 - 500 / 0 - 600
T1: 1d-space: -100 - 1000
T1: 1d-space: 0 - 100
T1: 2d-space: -200 - 2000 / -300 - 3000
T1: 2d-space: 0 - 200 / 0 - 300
T1: 3d-space: 0 - 400 / 0 - 500 / 0 - 600
T1: 3d-space: 0 - 400 / 0 - 500 / 0 - 600
T2: 1d-space: -100 - 1000
T2: 1d-space: 0 - 100
T2: 2d-space: -200 - 2000 / -300 - 3000
T2: 2d-space: 0 - 200 / 0 - 300
T2: 3d-space: 0 - 400 / 0 - 500 / 0 - 600
T2: 3d-space: 0 - 400 / 0 - 500 / 0 - 600
T3: 1d-space: -100 - 1000
T3: 1d-space: 0 - 100
T3: 2d-space: -200 - 2000 / -300 - 3000
T3: 2d-space: 0 - 200 / 0 - 300
T3: 3d-space: 0 - 400 / 0 - 500 / 0 - 600
T3: 3d-space: 0 - 400 / 0 - 500 / 0 - 600
